src/components/Auth.vue

```diff
@@ -0,0 +1,13 @@
+<template lang="html">
```

Comment:
This file is the component that is called into App.vue to deliver login features